### ISSUE TITLE
ComputerCraft is not completely optional

### ISSUE THREAD

**Issue by latibro** [latibro/Automation-Mod#3]
Seems like there is a bug in 0.1.0-alpha.3 where ComputerCraft is not completely optional, and the mod fails to load it ComputerCraft is not present

**Comment by latibro** [comment]
Stack trace it CC or CC:T is not installed:

```
java.lang.NoClassDefFoundError: latibro/automation/integration/computercraft/CCPeripheralTrait
at latibro.automation.interfacebox.InterfaceBoxBlock.func_149915_a(InterfaceBoxBlock.groovy:36)
at net.minecraft.block.Block.createTileEntity(Block.java:1274)
at net.minecraft.world.chunk.Chunk.func_177436_a(Chunk.java:623)
at net.minecraft.world.World.func_180501_a(World.java:343)
at net.minecraft.item.ItemBlock.placeBlockAt(ItemBlock.java:180)
at net.minecraft.item.ItemBlock.func_180614_a(ItemBlock.java:54)
at net.minecraft.item.ItemStack.func_179546_a(ItemStack.java:187)
at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:467)
at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1579)
at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2254)
at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:2020)
at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1808)
at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1098)
at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
at net.minecraft.client.main.Main.main(SourceFile:123)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.ClassNotFoundException: latibro.automation.integration.computercraft.CCPeripheralTrait
at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
... 21 more
Caused by: java.lang.NoClassDefFoundError: dan200/computercraft/api/peripheral/IPeripheral
at java.lang.ClassLoader.defineClass1(Native Method)
at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
... 23 more
Caused by: java.lang.ClassNotFoundException: dan200.computercraft.api.peripheral.IPeripheral
at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
... 27 more
Caused by: java.lang.NullPointerException
```

**Comment by latibro** [comment]
Remember to check if OC and IR has a simular problem that they are not completely optional yet

**Comment by latibro** [comment]
Fixed in 0.1.0-alpha.8
